views/appView.js

```diff
@@ -0,0 +1,55 @@
+var AppView = Backbone.View.extend({
+  el: 'body',
+
+  collection: VideosCollection,
+
+  events: {
+    'click #search-submit': 'search',
+  },
+
+  initialize: function () {
+    this.listenTo(this.model, 'change', this.renderVideo);
+  },
+
+  // search: function () {
+  //   console.log('clicked submit');
+  //   var searchQuery = this.$el.find('#search-input').val();
+  //   console.log('the search query is: ' + searchQuery);
+  //   // return searchQuery;
+  //   //
+  //   videosCollection.fetch({reset: true});
+  //   // videosCollection.parse(searchQuery);
+  //   // videosCollection.on('change', function () {console.log(videosCollection.toJSON());}); //change to what target function?
+  //
+  //   // this.makeCollection(searchQuery);
+  // },
+
+  search: function () {
+    
+    let searchQuery = $('#search-input').val();
+    var search = 'https://www.googleapis.com/youtube/v3/search?key=AIzaSyCIF-nV9qLeAriwePo8cTdgHGEuH_VAno0&part=snippet&type=video&q='+searchQuery;
+
+    this.model.get('videos').url = search;
+
+    this.model.get('videos').fetch({
+      success: function () {
+        appModel.set('searchQuery', searchQuery)
```

Comment:
Why would you need to wait until the fetch completed to set the search query.. wouldn't you want to do that before you fetch?